### PR TITLE
update to system.json 'compatibility' key for V11

### PR DESCRIPTION
**Pull request by mus0u**
with V11 i got an error:
`The system "swnr" is using the old flat core compatibility fields which are deprecated in favor of the new "compatibility" object`

i believe this will fix it.

### CODE REVIEW

File: src/system.yml [hunk 1]
```diff
@@ -212,15 +212,14 @@ packs:
     module: swnr
     name: one-roll-backing
     path: packs/one-roll-backing.db
-    system: swnr    
+    system: swnr
```
Comment (mus0u) [inline src/system.yml:215]: looks like emacs caught some stray whitespace, this is unrelated. happy to remove if desired

**Comment by wintersleepAI** [comment]
This should be fixed in the next release, thanks! It should be defined with YAML (you were using a JSON as a string which works but is not ideal).
